file.go

```diff
@@ -0,0 +1,42 @@
+package python
```

Comment:
please add license header.

Reply:
Sorry, I haven't noticed any license headers in other files.

Reply:
ah! my bad.
`go-python` was one of my first Go project. I didn't have a very clear open-sourcing procedure.

please ignore this. I'll adapt go-python to my current standard :)